### PR TITLE
Adding css to report

### PR DESCRIPTION
**Pull request by laUmar123**

### CODE REVIEW

File: database/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Database
+This folder is responsible for handling the database aspect of the project. It includes things like an sql file for defining and seeding the database,
```
Comment (fm1psy) [inline database/README.md:2]: Did you forget to finish your sentence? Go back and look over this please.

File: weekly-report/main.py [hunk 2]
```diff
@@ -103,32 +127,117 @@ def handler(event=None, context=None):
...
+    pie_html = convert_altair_chart_to_html_embed(two_layer_pie)
+
+    total_earthquakes = get_total_number_earthquakes(con)
+    highest_magnitude = get_top_magnitude_earthquake(con)[1]
+    most_significant_earthquake = get_top_significant_earthquakes(con)[0][1]
     logging.info("Altair charts converted to html")
 
+    prev_monday = previous_monday(CURRENT_DATE).strftime("%B %d, %Y")
+
     html_report = f"""
```
Comment (fm1psy) [inline weekly-report/main.py:143]: Good job on the html, I really like the detail you have included and the formatting you have done

**Review by fm1psy — CHANGES_REQUESTED** [review]
Just a small thing for you to look over.

**Review by joe1606 — APPROVED** [review]
Looks good, a suggestion would be to add tests if you have the time

**Pull request merged**